src/main/java/fr/delthas/skype/NotifConnector.java

```diff
@@ -675,7 +675,9 @@ public synchronized void disconnect() {
   }
   
   private synchronized void sendPacket(String command, String parameters, String body) throws IOException {
-    String headerString = registration != null ? "Registration: " + registration + "\r\n" : "";
+//    String headerString = registration != null ? "Registration: " + registration + "\r\n" : "";
+    // Weird, but it's working better without sending the registration token. Solved the 911 problem but needs more testing.
```

Comment:
This is suprising indeed, maybe they removed recently like they removed Challenge. Let's include this change for now if its works for you.